src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/ECCurve.cs

```diff
@@ -142,7 +142,21 @@ public static ECCurve CreateFromValue(string oidValue)
 
         private static ECCurve CreateFromValueAndName(string oidValue, string oidFriendlyName)
         {
-            return ECCurve.Create(new Oid(oidValue, oidFriendlyName));
+            Oid oid = null;
+
+            if (oidValue is null && oidFriendlyName is object)
```

Comment:
According to sharplab.io; the JIT emits identical code for this and `if (oidValue == null && oidFriendlyName != null)` (and the compiler emits identical IL).

Could we just stick with classic C# that doesn't make me have to think about what nuances `is object` carries with it? (Clearly the answer is "none when the type is known"; but it's still not a style that has been used at all in this assembly)